universe/manifest.tt

```diff
@@ -0,0 +1,76 @@
+class @vikram.hello
+#_[thingpedia_name="Test 1"]
+#_[thingpedia_description="Test 1 for Movies"]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="media"]
+{
+    import loader from @org.thingpedia.v2();
+    import config from @org.thingpedia.config.none();
+    import config from @org.thingpedia.config.none(api_key=$?);
+
+    entity movie #_[description = "Movie Information"];
+    entity actor #_[description = "People in movie"];
+    monitorable list query movie(   out id: Entity(vikram.hello:movie)
+                                    #_[canonical={
+                                        default = "base",
+                                        base = ["movie", "play", "film"],
+                                        passive_verb = ["named #", "titled #", "called #"],
+                                        property = ["name #", "title #"]
+                                    }],
+                                    out description : String
+                                    #_[canonical={
+                                        default = "base",
+                                        base = ["description", "overview", "summary"],
+                                        passive_verb=["summarized"]
+                                    }],
+                                    out release_date : Date
+                                    #_[canonical={
+                                        default="passive_verb",
+                                        base = ["date", "release date", "release year", "release day"],
+                                        property=["release date #"],
+                                        preposition=["from"],
+                                        passive_verb=["released in #", "filmed #", "released #"],
+                                        adjective_argmin = ["least recent", "oldest", "first"],
+                                        adjective_argmax = ["most recent", "latest", "newest", "now playing"],
+                                        passive_verb_argmax=["released most recently"],
+                                        passive_verb_projection=['released | on', 'filmed | on'],
+                                    }],
+                                    out rating : String
```

Comment:
You might want this to be an Enum of possible ratings, for additional robustness in natural language - that is, if you use an enum, you can ask for a movie "suitable for children" to mean "rated G" without mentioning the rating explicitly.